typescript/infra/scripts/xerc20/xerc20wl-add-bridge.ts

```diff
@@ -0,0 +1,83 @@
+import chalk from 'chalk';
+
+import {
+  LogFormat,
+  LogLevel,
+  configureRootLogger,
+  rootLogger,
+} from '@hyperlane-xyz/utils';
+
+import {
+  addBridgeToChainWL,
+  deriveWLBridgesConfig,
+  getAndValidateBridgesToUpdate,
+  getWarpConfigsAndArtifacts,
+} from '../../src/xerc20/utils.js';
+import {
+  getArgs,
+  withChains,
+  withDryRun,
+  withWarpRouteIdRequired,
+} from '../agent-utils.js';
+import { getEnvironmentConfig } from '../core-utils.js';
+
+// Designed to work with Wonderland variant of xERC20: https://github.com/hyperlane-xyz/xERC20
+async function main() {
+  configureRootLogger(LogFormat.Pretty, LogLevel.Info);
+  const { environment, warpRouteId, chains, dryRun } = await withChains(
+    withWarpRouteIdRequired(withDryRun(getArgs())),
+  ).argv;
+
+  const { warpDeployConfig, warpCoreConfig } =
+    getWarpConfigsAndArtifacts(warpRouteId);
+
+  const envConfig = getEnvironmentConfig(environment);
+  const multiProtocolProvider = await envConfig.getMultiProtocolProvider();
+  const envMultiProvider = await envConfig.getMultiProvider();
+
+  const bridgesConfig = await deriveWLBridgesConfig(
+    warpDeployConfig,
+    warpCoreConfig,
+    envMultiProvider,
+  );
```

Comment:
maybe you could consider deriving the limits config only for the chains specified in the command if any? it might speed things up if we only want to update on 1 or 2 chains and the route is big